01.data_download.py

```
@@ -61,21 +61,20 @@
+        if i['C1_NM'] == '주택담보대출':
             month_list.append(i['PRD_DE'])
             loanrate_list.append(i['DT'])
 
     loanrate_df = pd.DataFrame({'month': month_list, 'loanrate': loanrate_list})
     loanrate_df.to_csv(loanrate_path, index=False)
 
-        
-# display(loanrate_df)
-# display(py_json[:5])
+#%%
+display(loanrate_df)
```

Ruff (0.11.9)

Undefined name `display`

(F821)